test/services/ConfirmStakeIntent/confirmStakeIntent.test.ts

```diff
@@ -0,0 +1,84 @@
+import * as sinon from 'sinon';
+import ConfirmStakeIntentService from '../../../src/services/ConfirmStakeIntentService';
+import StubData from '../../test_utils/StubData';
+import assert from '../../test_utils/assert';
+import Repositories from '../../../src/repositories/Repositories';
+
+interface TestConfigInterface {
+  repos: Repositories;
+}
+
+let config: TestConfigInterface;
+
+const Web3 = require('web3');
+
+describe('ConfirmStakeIntentService.confirmStakeIntent()', () => {
```

Comment:
I think, this method should be private and tested with update only.

Reply:
Done!